frontend/javascripts/oxalis/api/api_latest.ts

```diff
@@ -584,6 +585,23 @@ class TracingApi {
     );
   }
 
+  /**
+   * Gets a segment object within the referenced volume layer. Note that this object
+   * does not support any modifications made to it.
+   *
+   * @example
+   * const segment = api.tracing.getSegment(
+   *   3,
+   *   "volume-layer-id"
+   * );
+   * console.log(segment.groupId)
+   */
+  getSegment(segmentId: number, layerName: string): Segment {
+    const segment = getSegmentsForLayer(Store.getState(), layerName).get(segmentId);
```

Comment:
What happens if the layer name does not exist?

Reply:
an error will be thrown which I find acceptable.